frontend/src/components/Button.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
 import styled from 'styled-components'
 
-const Button = styled.button<{ state: string }>`
-  height: 2.2rem;
-  width: 10rem;
-  border-radius: 100rem;
-  font-size: 1rem;
+const Button = styled.button<{ state: string; width: string; size: string }>`
```

Comment:
help update types to exactly what it should be!
```suggestion
const Button = styled.button<{ state: 'primary' | 'secondary'; width: string; size: 'small' | 'regular' }>`
```